sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/internal/debug/TestSourceInfo.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.debug;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class TestSourceInfo {
```

Comment:
```suggestion
class SourceInfoTest {
```

Reply:
This applies to `TestMetricDescriptor`, `TestDebugConfig`, and the other `TestSourceInfo` as well.